docs/content/reference.md

```diff
@@ -39,15 +39,15 @@ There are a collection of constraint objects that some methods have access to. T
 ### Listener function arguments
 Listener functions have access to a set of arguments that may change based on the method which the function is passed to. Below is an explanation of the different arguments. The below table details the different arguments and the methods they'll be accessible in.
 
-| Argument  | Description  |
-| :--- | :--- |
+| Argument  | Method | Description  |
```

Comment:
🤔 The language is a bit confusing here for me... I had "listener" at first, but the other sections describe this column as a "method".

I believe "method" is right after some thinking? As I understand, "listeners" are actually "listener functions" and these are the functions that are called for certain methods. I'd love to learn more nuance on this though!

Reply:
A sentense like "`payload` is available for all app.* listeners" makes sense to me, so I don't see issues with "Listener" here. Also, the last two rows actually have "All listeners" for the column. That said, "method" works too and it should be understood as well, so I don't have strong opinions on the naming, but personally I am  down "Listener" here.

Reply:
Thanks! This is helpful and I'm glad to know both **can** make sense. I think that (and the use of "all listeners" below) was confusing me most 😉

I'll update this to "listeners" to avoid causing confusion within these tables!